src/main/resources/application.yaml

```diff
@@ -107,3 +107,5 @@ widget:
       localization:
         error-title: Es ist ein Fehler aufgetreten.
         error-text: Stellen Sie sicher, dass die BundesIdent App installiert ist und versuchen Sie es erneut.
+sentry:
+  traces-sample-rate: 0.3
```

Comment:
Sentry wird in `application-production.yaml` und `application-staging.yaml` jeweils schon konfiguriert. Vielleicht ist eine Unterscheidungen zwischen den Umgebungen auch sinnvoll? Dann könnte man das dort jeweils auch dazuschreiben.

Reply:
Guter Punkt. Auf Staging kann die Samplerate höher sein.